internal/host/hostcommands/install_cmd.go

```diff
@@ -287,7 +292,48 @@ func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inve
 		}
 	}
 
-	if !hasStaticNetwork {
+	// Check if Manufacturer is IBM/S390 and ProductName is not "KVM/Linux" (the case for z/VM and LPAR).
+	// If this is the case than we need to extract the necessary z/VM kargs and append them.
+	if inventory != nil &&
+		inventory.SystemVendor != nil {
+		log.Debugf("Check for SystemVendor and ProductName: %s:%s\n", inventory.SystemVendor.Manufacturer, inventory.SystemVendor.ProductName)
+	} else {
+		log.Debug("Check for SystemVendor and ProductName: <nil>:<nil>\n")
+	}
```

Comment:
Please extract `inventory != nil && inventory.SystemVendor != nil` to a named variable and reuse in the condition below

Reply:
Preferably the name should indicate what the combination of these conditions even means

Reply:
Done (var used)